yorkie/config.go

```diff
@@ -25,11 +25,13 @@ import (
 	"github.com/yorkie-team/yorkie/pkg/log"
 	"github.com/yorkie-team/yorkie/yorkie/backend"
 	"github.com/yorkie-team/yorkie/yorkie/backend/mongo"
+	"github.com/yorkie-team/yorkie/yorkie/metrics"
 	"github.com/yorkie-team/yorkie/yorkie/rpc"
 )
 
 const (
-	DefaultRPCPort = 9090
+	DefaultRPCPort     = 9090
+	DefaultMetricsPort = 2112
```

Comment:
Now it seems the time has come to clean up the ports that Yorkie uses. Does port 2112 have any special meaning?

Reply:
I used the port in the example. There is no special meaning. :-)
https://prometheus.io/docs/guides/go-application/